src/main/kotlin/lotto/OutputView.kt

```diff
@@ -0,0 +1,36 @@
+package lotto
+
+object OutputView {
+    fun displayTickets(
+        ticketCount: Int,
+        ticketsList: List<Ticket>,
+    ) {
+        println("You have purchased $ticketCount tickets.")
+        ticketsList.forEach { ticket ->
+            println(ticket.numbers.joinToString(prefix = "[", postfix = "]", separator = ","))
+        }
+    }
+
+    fun displayResults(
+        winStats: Map<Rank, Int>,
+        returnRate: String,
+    ) {
+        val text =
+"""
+    
+Winning Statistics
+------------------
+3 Matches (${formatNumber(Rank.FIFTH.prize)} KRW) - ${winStats[Rank.FIFTH]} tickets
+4 Matches (${formatNumber(Rank.FOURTH.prize)} KRW) - ${winStats[Rank.FOURTH]} tickets
+5 Matches (${formatNumber(Rank.THIRD.prize)} KRW) - ${winStats[Rank.THIRD]} tickets
+5 Matches + Bonus Ball (${formatNumber(Rank.SECOND.prize)} KRW) - ${winStats[Rank.SECOND]} tickets
+6 Matches (${formatNumber(Rank.FIRST.prize)} KRW) - ${winStats[Rank.FIRST]} tickets
+Total return rate is $returnRate (A rate below 1 means a loss)
+""".trimIndent()
+        println(text)
+    }
```

Comment:
Nice use of a multi-line string 👍
Is the string unindented for output formatting reasons?

If so, try indenting it like the rest of your logic and see how the output looks 😄 :
```suggestion
    fun displayResults(
        winStats: Map<Rank, Int>,
        returnRate: String,
    ) {
        val text = """
        Winning Statistics
        ------------------
        3 Matches (${formatNumber(Rank.FIFTH.prize)} KRW) - ${winStats[Rank.FIFTH]} tickets
        4 Matches (${formatNumber(Rank.FOURTH.prize)} KRW) - ${winStats[Rank.FOURTH]} tickets
        5 Matches (${formatNumber(Rank.THIRD.prize)} KRW) - ${winStats[Rank.THIRD]} tickets
        5 Matches + Bonus Ball (${formatNumber(Rank.SECOND.prize)} KRW) - ${winStats[Rank.SECOND]} tickets
        6 Matches (${formatNumber(Rank.FIRST.prize)} KRW) - ${winStats[Rank.FIRST]} tickets

        Total return rate is $returnRate (A rate below 1 means a loss)
    """.trimIndent()

        println(text)
    }
```

It would be great if you could look into what allows our multi-line string to maintain its formatting in the console, even when we follow proper indentation in the code.